src/olympia/editors/models.py

```diff
@@ -732,13 +732,13 @@ def calculate_weight(self):
                 AbuseReport.objects
                 .filter(Q(addon=addon) | Q(user__in=addon.listed_authors))
                 .filter(created__gte=one_year_ago).count() * 10, 100),
-            # Each "recent" review with a score of 3 or less adds 2 to the
-            # weight, up to a maximum of 100.
-            'negative_reviews': min(
+            # 1% of the total of "recent" reviews with a score of 3 or less
+            # adds 2 to the weight, up to a maximum of 100.
+            'negative_reviews': min(int(
                 Review.objects
                 .filter(addon=addon)
                 .filter(rating__lte=3, created__gte=one_year_ago)
-                .count() * 2, 100),
+                .count() / 100. * 2), 100),
```

Comment:
If you use float-notation, I'd prefer using float notation for the whole expression, e.g `.count() / 100.0 * 2.0` which makes it a bit easier to discover.